README.md

```diff
@@ -28,6 +28,10 @@ Please use the version selector at the top of the site to ensure you are using t
 
 If you encounter issues, review the [troubleshooting docs][30], [file an issue][4], or talk to us on the [#velero channel][25] on the Kubernetes Slack server.
 
+**NOTE**: The issues section in this repository is for tracking issues for upstream opensource velero.  If you encounter 
+any issues while using a downstream product that consumes velero, like OADP, TMC Data Protection, please make sure to file a
+ticket via the support channel of the product for the initial triage.
+
```

Comment:
Can you add links for OADP support?

https://access.redhat.com/support/cases/#/troubleshoot/describe-issue?product=OpenShift%20API%20for%20Data%20Protection&version=

Thanks!

Reply:
Hmm. Maybe a bullet list below the note for specific downstream products with known support ticket URLs would be appropriate here, using the above for OADP, and accepting contribution for other links as appropriate.

Reply:
yes.. IMHO ideally we collect the right link/pointer to popular downstream support of Velero and link them together.  In the hope users find the appropriate link and save upstream a bit of time and energy.

Reply:
And another link for FAQ: https://red.ht/oadp-faq